prisma/schema.prisma

```diff
@@ -309,3 +311,21 @@ model ResponseSubpart {
   UserResponse   UserResponse? @relation(fields: [userResponseId], references: [id])
   userResponseId String?       @db.ObjectId
 }
+
+model Comment {
+  id              String    @id @default(auto()) @map("_id") @db.ObjectId
+  content         String
+  likeNum         Int       @default(0)
```

Comment:
is it possible to also track who likes a comment?

for example, add a model for likes that includes commentid and userid?

this way, when users like a comment, they can remove the like later

Reply:
created the model CommentLike.
created api route: /api/comments/likes
created both post and delete function for users liking and unliking a comment.

Reply:
and also tested the url on postman